### PR TITLE
Fix docker build

### PR DESCRIPTION
**Pull request by muXxer**

### CODE REVIEW

**Review by lmoe — APPROVED** [review]

File: .github/workflows/docker-push.yml [hunk 1]
```diff
@@ -13,17 +13,15 @@ jobs:
       image_name: iotaledger/wasp
 
     steps:
-      - uses: actions/checkout@v3
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v3
+
+      #- name: Set up QEMU
+      #  uses: docker/setup-qemu-action@v2
```
Comment (jorgemmsilva) [inline .github/workflows/docker-push.yml:20]: qemu?

**Review by jorgemmsilva — APPROVED** [review]

File: .github/workflows/docker-push.yml (hunk 1 above)
Comment (alexsporn) [reply .github/workflows/docker-push.yml:20]: that is used when you want to also build arm64 versions of the containers

File: .github/workflows/docker-push.yml [hunk 2]
```diff
@@ -35,15 +33,22 @@ jobs:
             type=semver,pattern={{version}}
             type=semver,pattern={{major}}.{{minor}}
             type=semver,pattern={{major}}
+            type=match,pattern=v(\d+.\d+),suffix=-alpha,group=1,enable=${{ contains(github.ref, '-alpha') }}
+            type=match,pattern=v(\d+.\d+),suffix=-beta,group=1,enable=${{ contains(github.ref, '-beta') }}
+            type=match,pattern=v(\d+.\d+),suffix=-rc,group=1,enable=${{ contains(github.ref, '-rc') }}
```
Comment (alexsporn) [inline .github/workflows/docker-push.yml:38]: I would remove these, because Wasp is using the 0.x.y versioning right now, and not the 1.0.0-beta.X pattern

Comment (muXxer) [reply .github/workflows/docker-push.yml:38]: maybe they should switch to 1.0.0-beta.1 :D

**Pull request merged**